app/organization/roles/show/template.hbs

```diff
@@ -88,6 +63,31 @@
             </div>
           </div>
 
+          <div class="panel-section">
+            <form>
+              <label>Add member</label>
+              <div class="invite-user">
+                <button class="btn btn-primary" {{action "inviteUser" invitedUser}}>Add</button>
+                <div class="user-select-container">
+                  {{user-select users=nonmembers selected=invitedUser}}
+                </div>
+              </div>
+            </form>
+          </div>
+
+          <div class="panel-section">
+            <form role='form' {{action "inviteByEmail" inviteByEmail on="submit"}}>
+              {{server-validation-errors model=invitation}}
+              <label>Invite by email</label>
+              <div class="invite-user">
+                <button class="btn btn-primary"{{action "inviteByEmail" invitedEmail}}>Invite</button>
```

Comment:
`<button class="btn btn-primary"{{action "inviteByEmail" invitedEmail}}>Invite</button>`

^ should have a space before `{{action`